README.md

```diff
@@ -253,6 +254,76 @@ https://youtu.be/dQw4w9WgXcQ
 ></iframe>
 ```
 
+### Custom Transformers
+
+The plugin allows you to pass an array of custom transformers that will be
+executed additionally to the default ones.
+
+#### Usage
+
+```js
+// In your gatsby-config.js
+import someSiteTransformer from './src/some-site-transformer';
+
+plugins: [
+  {
+    resolve: `gatsby-transformer-remark`,
+    options: {
+      plugins: [
+        {
+          resolve: `gatsby-remark-embedder`,
+          options: {
+            customTransformers: [someSiteTransformer],
+          },
+        },
+      ],
+    },
+  },
+];
+```
+
+Each element of the array should be an object with two methods that receive the
+URL argument:
+
+- `shouldTransform(url)`
+- `getHTML(url)`
+
+The `shouldTransform` method should check if the URL matches the one intended to
+transform; it should to return a boolean value.
+
+The `getHTML` method is executed when the URL has been matched to transform. It
+should return the transformed HTML. This function can be asynchronous, either by
+marking it as `async` or by manually returning a `Promise` object.
+
+#### Example transformer object
+
+```js
+// myDomainTransformer.js
```

Comment:
Maybe rename this to `someSiteTransformer.js` so it's the same name as we use in `gatsby-config`? 🤔

Reply:
Sure, maybe to `some-site-transformer.js`, as we already reference it in the config example:

```javascript
// In your gatsby-config.js
import someSiteTransformer from './src/some-site-transformer';
```